src/PiGui.cpp

```diff
@@ -284,7 +284,7 @@ void *PiGui::makeTexture(const std::string &filename, unsigned char *pixels, int
 	Pi::renderer->CheckRenderErrors(__FUNCTION__, __LINE__);
 	// nasty bit as I invoke the TextureGL
 	Graphics::OGL::TextureGL *pGLTex = reinterpret_cast<Graphics::OGL::TextureGL*>(pTex);
-	Uint32 result = pGLTex->GetTexture();
+	Uint32 result = pGLTex->GetTextureID();
```

Comment:
I wonder if this is a more general question, should we migrate to C++11 standard, and avoid pioneer home made stuff? e.g. [RefCountedPtr to std::shared_ptr](https://pioneerspacesim.net/forum/viewtopic.php?f=3&t=267)